drivers/watchdog/wdt_wwdg_stm32.c

```diff
@@ -84,7 +90,7 @@ static u32_t wwdg_stm32_get_timeout(struct device *dev, u32_t prescaler,
 				    u32_t counter)
 {
 	u32_t divider = WWDG_INTERNAL_DIVIDER * (1 << (prescaler >> 7));
-	float f_wwdg = (float)wwdg_stm32_get_pclk(dev) / divider;
+	float f_wwdg = wwdg_stm32_get_pclk(dev) / divider;
```

Comment:
This change is not part of this issue.

Reply:
this is covered by the PullRequest 
driver: watchdog: Incorrect expression in wdt_wwdg_stm32.c [#20617]